### DISCUSSION TITLE
2007/11/29/the-red-light-is-just-a-suggestion

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#193] [Announcements]
# 2007/11/29/the-red-light-is-just-a-suggestion

最近接待了好几个老外同事。讨论起中国的出租司机，他们都是直摇头。不过，不跟他们聊还真不知道，原来“对红灯视而不见”可以这么表达：The red light is just a suggestion.

https://blog.qizhen.xyz/2007/11/29/the-red-light-is-just-a-suggestion

**Comment by ruanqizhen** [comment]
之前博客下的留言：

清扬
2007年12月8日 09:38 编辑
The red night is just a suggestion.没想到用英语说起来这么有意思~
不过感觉对红灯最视而不见的不是司机,而是行人
 

回复
sandy
2007年12月1日 04:35 编辑
The Red Light is the Law.

回复
